modules/invoicein/view/edit.php

```diff
@@ -202,8 +227,16 @@
 while($row2 = $_lib['db']->db_fetch_object($result2))
 {
     $LineID=$row2->LineID;
-    $sumline = round($row2->QuantityDelivered * $row2->UnitCustPrice, 2);
-    $vatline = round(($row2->Vat/100) * $sumline,2);
+    $sumline = $row2->QuantityDelivered * $row2->UnitCustPrice;
+    // if tax amount exists, use it, if not - calculate
+    if ($row2->TaxAmount != 0) {
+      $vatline = $row2->TaxAmount;
+    } else {
+      $vatline = ($row2->Vat/100) * $sumline;
+    }
```

Comment:
The above if could be changed to recalculate if it is a manually created invoice. Discussion needed.